addons/frag/functions/fnc_getFragInfo.sqf

```diff
@@ -0,0 +1,99 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Jaynus, NouberNou, Lambda.Tiger
+ * This function returns fragmentation parameters for a specific ammo type.
+ *
+ * Arguments:
+ * 0: Ammo classname <STRING>
+ *
+ * Return Value:
+ * _ammoInfo <ARRAY>
+ *  0: Search range for fragments in meters <NUMBER>
+ *  1: Gurney equation calculated speed <NUMBER>
+ *  2: Array of fragment types <ARRAY>
+ *  3: Mass of fragmenting metal modified for frag count <NUMBER>
+ *
+ * Example:
+ * "B_556x45_Ball" call ace_frag_fnc_getFragInfo
+ *
+ * Public: No
+ */
+
+params ["_ammo"];
+
+GVAR(fragInfoCache) getOrDefaultCall [
+    _ammo, {
+    private _ammoConfig = configFile >> "CfgAmmo" >> _ammo;
+    private _fragTypes = ACE_FRAG_DEFAULT_FRAG_TYPES;
+    private _notifyMissingEntries = false;
+    if (isArray (_ammoConfig >> QGVAR(classes))) then {
+        _fragTypes = getArray (_ammoConfig >> QGVAR(classes));
+    } else {
+        _notifyMissingEntries = true;
+    };
+
+    /************ Gurney equation notes *****************//*
+    * see https://en.wikipedia.org/wiki/Gurney_equations
+    *
+    * gurney_k is the geometry constant added to _metalMass/_chargeMass
+    * gurney_c = sqrt(2E)
+    *
+    * _chargeMass = 185; - grams of comp-b
+    * _metalMass = 210; - grams of metal are accelerated by explosion
+    * _geometryCoefficient = 3/5; - spherical K factor
+    * _gurneyConstant = 2843; - Gurney constant of comp-b in /ms
+    *
+    * _chargeMass = 429; - grams of tritonal
+    * _metalMass = 496; - grams of metal are accelerated by explosion
+    * _geometryCoefficient = 1/2; - cylindrical K factor
+    * _gurneyConstant = 2320; - Gurney constant of tritonal in m/s
+    * Equation - 0.8 for empirical 80% speed
+    * 0.8 * (((_metalMass / _chargeMass) + _geometryCoefficient) ^ - (1 / 2)) * _gurneyConstant;
+    * or 0.8 * _gurneyConstant * sqrt (_chargeMass /(_metalMass + _chargeMass * _geometryCoefficient)); (slightly faster to compute)
```

Comment:
If the latter is faster, then we shouldn't even give a slower option, knowing ppl.

Hell, although I'm absolutely 1 of the worst to talk math, the latter even looks simpler.
```suggestion
    * 0.8 * _gurneyConstant * sqrt (_chargeMass / (_metalMass + _chargeMass * _geometryCoefficient));
```

Reply:
It's honestly mainly to make it clear when you look at the Wikipedia that the same equation is used here and on line 93. The later is an algebraic equivalent, but if you're bad at math or maybe don't take the moment to do the algebra you'd think that line 93 is incorrect simply because it is different.

Reply:
I don't doubt the validity (ACE already uses such math in many places I'm happy to have heard of), but if one is faster, then that's the one to use, and in this case it happens, as you said, the latter is faster and also looks simpler. If the other one is needed for clarity like you said, then it should also be included.